lib/cocoapods/installer/user_project_integrator/target_integrator.rb

```diff
@@ -264,7 +264,22 @@ def add_copy_resources_script_phase
             output_paths = []
             unless resource_paths_by_config.values.all?(&:empty?)
               input_paths = [target.copy_resources_script_relative_path, *resource_paths_by_config.values.flatten.uniq]
-              output_paths = ['${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}']
+
+              output_paths = resource_paths_by_config.values.flatten.uniq.map do |input_path| 
+                base_path = '${TARGET_BUILD_DIR}/${UNLOCALIZED_RESOURCES_FOLDER_PATH}'
+                extname = File.extname(input_path)
```

Comment:
should probably extract this in a method which will be easier to test for each extension

Reply:
Would you prefer it in the same class or somewhere else?

Reply:
could be under `private` section within the same class. To make it easier for testing you can either use `send` or just not make it private method, shouldnt matter that much.